ubiquity-fundraising-research-april-may-2024.md

```diff
@@ -0,0 +1,87 @@
+
+Questions arise regarding the Ubiquity Dollar about mechanisms for maintaining stability, issuance, prevalence, and regulation. Most experts noted that most tokens launched in 2024 are losing their value, and they would like to see a plan for investor exits or for maintaining the value of UBQ.
+
+## Team
+Not counting Sergey, the Ubiquity team is anonymous, which is common for crypto projects. This does not significantly add or reduce investor trust. From the perspective of public promotion, it is necessary to have influencers who will be the voice of the project. Typically, investors want to see a cohesive team because companies with solo entrepreneurs are less stable.
+
+### GitHub
+Most experts highly rated Ubiquity's GitHub. A large number of actively discussed repositories distinctly distinguishes Ubiquity from many projects. One expert suggested comparing the activity with the repositories of popular crypto projects, in particular, Eigen Foundation [https://github.com/eigenfoundation](https://github.com/eigenfoundation).
+
+## Market and Competition
+In its materials, Ubiquity has not clearly shown which market it serves and what problems it solves. Listed problems and solutions do not distinguish Ubiquity from other products. The competitive environment for Ubiquity is incorrectly defined. For each of Ubiquity's product lines, it is necessary to define the competitive environment. Companies described as competitors based on the AI platform criterion are not direct competitors because Ubiquity is not an AI platform. An analysis found no AI products and services offered by Ubiquity. Using LLM via API does not place it alongside companies that train their own models or create solutions using complex prompts. Experts see a wave of projects trying to use the AI + crypto theme. If a project lacks AI competence like Ubiquity, such positioning is perceived negatively.
+
+Similar systems to those offered by Ubiquity are not popular on GitHub 
+
+[https://github.com/marketplace?type=apps&category=project-management](https://github.com/marketplace?type=apps&category=project-management) 
+
+However, similar systems exist, and some of them have more installations than Ubiquity:
+
+- Feature bot
+
+[https://github.com/marketplace/feature-bot](https://github.com/marketplace/feature-bot)
+
+[https://github.com/feature-sh](https://github.com/feature-sh)
+
+[https://feature.sh/](https://feature.sh/)
+
+[https://docs.feature.sh/](https://docs.feature.sh/)
+
+
+## Products and Users
+
+### Users
+Experts note the near absence of users and partners, which does not provide an opportunity to assess traction.
```

Comment:
This is the main pushback I received from my conversations. We will need to onboard essentially any remote organization to boost our "vanity metrics"